frontend/src/components/StatusTag.vue

```diff
@@ -187,7 +203,34 @@ export default {
       }
 
       return `${name} [${errorState}${since}]`
+    },
+    conditionFromType (type) {
+      let condition = knownConditions[type]
+      if (condition) {
+        return condition
+      }
+      let displayNameComponents = []
+      let shortNameComponents = []
+      const conditionPattern = /[A-Z]*([A-Z])[a-z]*/g
+      let conditionComponent
+      while ((conditionComponent = conditionPattern.exec(type)) !== null) {
+        displayNameComponents.push(conditionComponent[0])
+        shortNameComponents.push(conditionComponent[1])
+      }
+      if (shortNameComponents.length > 1) {
+        // Remove last component as it is usually not part of the condition name (e.g. availability)
+        shortNameComponents = shortNameComponents.slice(0, shortNameComponents.length - 1)
+        displayNameComponents = displayNameComponents.slice(0, displayNameComponents.length - 1)
+      }
+
+      const displayName = join(displayNameComponents, ' ')
+      const shortName = join(shortNameComponents, '')
+      knownConditions[type] = { displayName, shortName } // cache
+      return knownConditions[type]
     }
+  },
+  beforeMount () {
+    merge(knownConditions, this.cfg.knownConditions)
```

Comment:
move knownConditions to store and rename known to wellKnown?

Reply:
done